src/pkg/Directory.Build.targets

```diff
@@ -14,11 +14,13 @@
       Condition="
         '$(ReleaseBrandSuffix)' == '' and
         '$(PreReleaseVersionLabel)' != '' and
-        $(PreReleaseVersionLabel.StartsWith('preview'))">
-      <!-- Convert 'preview7' to 'Preview 7'. -->
+        '$(PreReleaseVersionIteration)' != ''">
+      <!-- Convert 'preview.7' to 'Preview 7'.
+          'preview' will come from the pre-release version iteration and the numeric value
+          will be the PreReleaseVersionIteration. -->
+
       <ReleaseBrandSuffix>$(PreReleaseVersionLabel.Substring(0,1).ToUpperInvariant())</ReleaseBrandSuffix>
-      <ReleaseBrandSuffix>$(ReleaseBrandSuffix)$(PreReleaseVersionLabel.Substring(1,6))</ReleaseBrandSuffix>
```

Comment:
This is still necessary with `$(PreReleaseVersionLabel.Substring(1))` or else this is e.g. `P 7` (`A 1`).